presto-iceberg/pom.xml

```diff
@@ -222,6 +222,13 @@
             <groupId>io.prestosql</groupId>
             <artifactId>presto-testing</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <!-- Exclude caffeine (dependency of JDBI) as it conflicts with org.apache.iceberg dependencies -->
```

Comment:
Since this conflicts with iceberg-core, this means that lib is not correctly shaded. Let's create an issue for that and link it here & and in https://github.com/prestosql/presto/issues/1324

Reply:
https://github.com/prestosql/presto/issues/1324

Reply:
thanks